### PR TITLE
GCP update

### PR DESCRIPTION
**Pull request by pdros-splunk**
- Closed issue with this gcp_update branch about Python installation, if there is no Python3 already installed on a machine, the script will do this automatically 
- Added openssl11 installation which helps during dealing with GCP

### CODE REVIEW

File: vars.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 soar_download_url: <DOWNLOAD URL GOES HERE>
 soaruser_password: <soaruser PASSWD GOES HERE>
+soar_version: <SOAR version in Numeric>
```
Comment (ricardom-splunk) [inline vars.yaml:3]: Can we have an example of the format? As a comment in the file...

File: fdse_soar_installer.yaml [hunk 2]
```diff
@@ -84,20 +87,71 @@
...
+    
+    - name: Install python and pip3
       become: yes
       become_user: root
-      ansible.builtin.pip:
+      ansible.builtin.yum:
         name:
-          - passlib
-          - rpdb
+          - python36
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:136]: newer versions are not in the yum repo by default... but would it be worth trying to install something newer?

File: fdse_soar_installer.yaml [hunk 3]
```diff
@@ -84,20 +87,71 @@
...
+          - libselinux-python
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:137]: what is this for? didn't we disable SELinux on the first step? or is this something else?

File: fdse_soar_installer.yaml [hunk 4]
```diff
@@ -84,20 +87,71 @@
         owner: soaruser
         mode: '0755'
 
+    - name: check openssl existance
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:90]: existEnce

File: fdse_soar_installer.yaml [hunk 5]
```diff
@@ -84,20 +87,71 @@
         owner: soaruser
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:87]: As the installation for SOAR 6 already suggests creating the user `phantom`, maybe we should change the name here to phantom, for consistency?

File: README.md [hunk 6]
```diff
@@ -46,17 +46,9 @@ This ansible playbook installs Splunk SOAR on remote machines running Centos7.
 		- Hint! - remember that token for link expired after ~10min
 	- Paste this link to soar_download_url
 	- Enter password
+	- Enter version of SOAR as a Numeric value
```
Comment (ricardom-splunk) [inline README.md:49]: Either here and/or as a comment on the file, be more specific about the format of the number

File: fdse_soar_installer.yaml (hunk 3 above)
Comment (pdros-splunk) [reply fdse_soar_installer.yaml:137]: Yeah, libselinux can be deleted, I have missed it

File: fdse_soar_installer.yaml (hunk 5 above)
Comment (pdros-splunk) [reply fdse_soar_installer.yaml:87]: I guess the question is if anyone pays attention to it. It's just a proposed name. If someone is using an ansible script then it won't even recognise that someone is creating a new user named phantom or soaruser

File: README.md (hunk 6 above)
Comment (pdros-splunk) [reply README.md:49]: It is not a problem to add

File: fdse_soar_installer.yaml (hunk 2 above)
Comment (pdros-splunk) [reply fdse_soar_installer.yaml:136]: I think not because soar installer will update python to 3.9

File: fdse_soar_installer.yaml [hunk 7]
```diff
@@ -58,16 +58,16 @@
       become: yes
       become_user: root
       ansible.builtin.user:
-        name: soaruser
+        name: phantom
```
Comment (pdros-splunk) [inline fdse_soar_installer.yaml:61]: I decided to change the user to phantom, let's be consistent with normal installation way

**Pull request merged**